proto/cv/scanner/v1/scanner.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package cv.scanner.v1;
```

Comment:
Why adding the extra `scanner` ?

Reply:
This is kind of nice since it is namespace-service-version. And we keep that in the repo